runelite-client/src/main/java/net/runelite/client/plugins/attackstyles/AttackStylesPlugin.java

```diff
@@ -191,6 +212,17 @@ public void onAttackStyleChange(VarbitChanged event)
 		}
 	}
 
+	@Subscribe
+	public void onAutoRetaliateChange(VarbitChanged event)
```

Comment:
Actually, there is already varbitChanged, multiple times in this class. Merge all methods to 1 VarbitChanged method

Reply:
You want me to merge all these onXChange events into one when that was already in place? Explain.